spring-integration-mqtt/src/main/java/org/springframework/integration/mqtt/outbound/AbstractMqttMessageHandler.java

```diff
@@ -134,7 +134,7 @@ public boolean isRunning() {
 
 	@Override
 	protected void handleMessageInternal(Message<?> message) throws Exception {
-		this.connectIfNeeded();
+		connectIfNeeded();
```

Comment:
Interesting; STS doesn't complain about this usage; nor does `javac`.

IDEA bug?